### PR TITLE
creation of workflow for plotting of contig/scaffold sizze and Nx for…

### PR DESCRIPTION
**Pull request by Delphine-L**
… one and 2 assemblies using gfastats outputs

### CODE REVIEW

**Comment by mvdbeek** [comment]
Can you add the missing metadata ? https://github.com/galaxyproject/iwc/blob/main/workflows/README.md#add-required-metadata

**Review by wm75 — COMMENTED** [review]
I think you can shorten the single output WF significantly by:

- making use of the latest Compute version
- replacing tail/cut/param value/add column with datamatsh/param value/compose expression

see my demo at https://usegalaxy.eu/u/wolfgang-maier/w/plot-gfastats-revised

This will also save a column in the computed table and you could save two more by making use of the new Compute tools Replace action instead of having it Append new columns.


Haven't looked at the two outputs WF, but maybe it can do similar things?

File: workflows/VGP-assembly-v2/VGP-Plot_gfastats_pair/Plot-gfastats-output---2-assemblies.ga [hunk 1]
```diff
@@ -0,0 +1,1576 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
+    "format-version": "0.1",
+    "release": "0.1",
+    "name": "Plot gfastats output - 2 assemblies",
+    "steps": {
```
Comment (simleo) [inline workflows/VGP-assembly-v2/VGP-Plot_gfastats_pair/Plot-gfastats-output---2-assemblies.ga:7]: `license` and `creator` are missing

File: workflows/VGP-assembly-v2/VGP-Plot_gfastats/Plot-gfastats-output.ga [hunk 2]
```diff
@@ -0,0 +1,604 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
+    "format-version": "0.1",
+    "release": "1.0",
+    "name": "Plot gfastats output",
```
Comment (simleo) [inline workflows/VGP-assembly-v2/VGP-Plot_gfastats/Plot-gfastats-output.ga:6]: `license` and `creator` are missing

**Pull request closed without merging**